dirty-lib.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/actcat/sideci_go_sandbox/lib"
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/naoina/denco"
+	_ "github.com/naoina/kocha"
+	_ "github.com/revel/revel"
+	_ "github.com/zenazn/goji"
+)
+
+func HttpHandler(w http.ResponseWriter, r *http.Request, p denco.Params) {
```

Comment:
__[golint]__
* [golint] exported function HttpHandler should have comment or be unexported
* [golint] func HttpHandler should be HTTPHandler

<!-- Comment by [SideCI](https://sideci.com) -->